### PR TITLE
Dynamo Connector Updation

### PR DESCRIPTION
**Pull request by RounakDhillon**
### Describe your changes:

Fixes <issue-number>

I worked on updating data profiler in Dynamo Connector.

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ slug: /connectors/database/greenplum
 name="Greenplum"
 stage="BETA"
 platform="OpenMetadata"
-availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt"]
+availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", , "view Lineage"]
```
Comment (SumanMaharana) [inline openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md:10]: we have an extra comma here

File: openmetadata-docs/content/v1.5.x/connectors/database/mysql/index.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ slug: /connectors/database/mysql
 name="MySQL"
 stage="PROD"
 platform="OpenMetadata"
-availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "View Lineage", "View Column-level Lineage"]
-unavailableFeatures=["Query Usage", "Owners", "Tags", "Stored Procedures"]
+availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "View Lineage", "View Column-level Lineage, "Query Usage"]
```
Comment (SumanMaharana) [inline openmetadata-docs/content/v1.5.x/connectors/database/mysql/index.md:10]: quotes are not closed for  `"View Column-level Lineage`

**Review by SumanMaharana — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md (hunk 1 above)
Comment (RounakDhillon) [reply openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md]: fixed

File: openmetadata-docs/content/v1.5.x/connectors/database/mysql/index.md (hunk 2 above)
Comment (RounakDhillon) [reply openmetadata-docs/content/v1.5.x/connectors/database/mysql/index.md]: fixed

File: openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ slug: /connectors/database/greenplum
 name="Greenplum"
 stage="BETA"
 platform="OpenMetadata"
-availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt"]
+availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "view Lineage"]
```
Comment (ayush-shah) [inline openmetadata-docs/content/v1.5.x/connectors/database/greenplum/index.md:10]:
```suggestion
availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "View Lineage"]
```

**Pull request closed without merging**